pkgs/racket-doc/scribblings/reference/contracts.scrbl

```diff
@@ -1864,14 +1864,24 @@ is provided from the module. In
 addition, clients of the module must live up to the contract specified
 by @racket[contract-expr] for each export.
 
-The @racket[contract-out] form treats modules as units of
-blame. The module that defines the provided variable is expected to
-meet the positive (co-variant) positions of the contract. Each module
-that imports the provided variable must obey the negative
-(contra-variant) positions of the contract. Each @racket[contract-expr]
-in a @racket[contract-out] form is effectively moved to the end of the
-enclosing module, so a @racket[contract-expr] can refer to variables
-that are defined later in the same module.
+@margin-note{Technically, the @racket[contract-out] form treats modules as
+units of blame. The module that defines the provided variable is expected to
+meet the positive (co-variant) positions of the contract. Each module that
+imports the provided variable must obey the negative (contra-variant)
+positions of the contract.}
+The contracts of @racket[contract-out] are placed at the boundary of the
+enclosing module. This means that, for example, if the module provides a form
+@racketid[my-form] with @racket[contract-out], the contract is checked only
+when @racketid[my-form] is called from outside the module, but when called
+from within the module. When @racketid[my-form] is called from outside the
```

Comment:
Was this supposed to be “but **not** when called from within the module”?

Reply:
Thanks for catching this. I thought I had already fixed it, but I forgot to push :/